Makefile

```diff
@@ -23,11 +23,12 @@ check:
 	find . -name "*.pyc" -exec rm {} \;
 	PYTHONPATH=$(CURDIR) PYTHONDONTWRITEBYTECODE=1 python3 -m pytest --color=$(COLOR) --verbose --showlocals --cov=packit_service --cov-report=$(COV_REPORT) $(TEST_TARGET)
 
-test_image: files/install-deps-worker.yaml files/install-deps.yaml files/recipe-tests.yaml
+build-test-image: files/install-deps-worker.yaml files/install-deps.yaml files/recipe-tests.yaml
 	$(CONTAINER_ENGINE) build --rm -t $(TEST_IMAGE) -f files/docker/Dockerfile.tests --build-arg SOURCE_BRANCH=$(SOURCE_BRANCH) .
 
-check_in_container:
+check-in-container:
 	@# don't use -ti here in CI, TTY is not allocated in zuul
+	@# Note: once in container, 'make check' (else recursive check-in-container)
```

Comment:
I would say this is self-explanatory, so there is no need to comment about it (i.e: The comment doesn't communicate any information that cannot be easily figure out just reading the lines bellow)

Reply:
I removed the offending comments from Makefile.